src/sentry/db/models/fields/bounded.py

```diff
@@ -64,12 +64,8 @@ class BoundedBigAutoField(models.AutoField):
 
         MAX_VALUE = 9223372036854775807
 
-        def db_type(self, connection):
-            engine = connection.settings_dict["ENGINE"]
-            if "postgres" in engine:
-                return "bigserial"
-            else:
-                raise NotImplementedError
+        def db_type(self):
```

Comment:
@wedamija I'm not sure what uses these functions - do you know if it's safe to remove a positional arg from this signature?

Reply:
I'd leave the signature the same, likely will break things otherwise.

Reply:
Oh haha you're right:

```
  File "/Users/joshua.li/.local/share/virtualenvs/sentry2/lib/python2.7/site-packages/django/db/models/fields/__init__.py", line 636, in db_parameters
    type_string = self.db_type(connection)
TypeError: db_type() takes exactly 1 argument (2 given)
```